src/renderers/noop/ReactNoop.js

```diff
@@ -146,6 +149,10 @@ var ReactNoop = {
     ReactNoop.flushLowPri();
   },
 
+  performHighPriWork(fn: Function) {
+    NoopRenderer.performWithPriority(HighPriority, fn);
```

Comment:
I'm not sure we want to expose this full granularity in the API we expose to renderers. I've been thinking about a different API for scheduling that would be based on expiration date buckets. If we don't expose it to users, maybe we also shouldn't expose it to renderers?